CONTRIBUTING.md

```diff
@@ -150,45 +150,69 @@ The TiKV team actively develops and maintains a bunch of dependencies used in Ti
 
 See more in [TiKV Community](https://github.com/tikv/community).
 
-### Format of the commit message
+### Linking issues
+
+Code repositories in TiKV community require **ALL** the pull requests referring to its corresponding issues. In the pull request body, there **MUST** be one line starting with `Issue Number: ` and linking the relevant issues via the [keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword), for example:
 
-We follow a rough convention for commit messages that is designed to answer two
-questions: what changed and why. The subject line should feature the what and
-the body of the commit should describe the why.
+If the pull request resolves the relevant issues, and you want GitHub to close these issues automatically after it merged into the default branch, you can use the syntax (`KEYWORD #ISSUE-NUMBER`) like this:
 
 ```
-engine/raftkv: add comment for variable declaration.
+Issue Number: close #123
+```
 
-Improve documentation.
+If the pull request links an issue but does not close it, you can use the keyword `ref` like this:
 
-Close #1234.
+```
+Issue Number: ref #456
 ```
 
-The format can be described more formally as follows:
+Multiple issues should use full syntax for each issue and separate by a comma, like:
 
 ```
-<subsystem>: <what changed>
-<BLANK LINE>
-<why this change was made>
-<BLANK LINE>
-<issue reference>
-<BLANK LINE>
-Signed-off-by: <Name> <email address>
+Issue Number: close #123, ref #456
```

Comment:
Is it supported by Github?

Reply:
`ref` keyword is not supported by GitHub. Multiple issues format follows GitHub's convention. It's documented in https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword.